lib/common/model.js

```diff
@@ -553,6 +636,106 @@ function ModelFactory (
                 $pullAll: values
             };
             var options = {new: true};
+            return self.updateAndFindMongo(query, update, options);
+        },
+
+
+        /**
+         * Using an Id, find document and add one or more items 
+         * from a list of this document, the values provided specifies the list and items. 
+         * The signs is an object list, each sign related to one or more object items in [value].
+         * The sign means primer key of object for determining if the object exist in list already.
+         * Returns a waterline (bluebird) promise with the document find and modify
+         *
```

Comment:
Could you please elaborate more on this description? or using a simple example? It is a little difficult to understand.